pkg/controller/grafanadashboard/dashboard_controller.go

```diff
@@ -288,40 +302,29 @@ func (r *ReconcileGrafanaDashboard) reconcileDashboards(request reconcile.Reques
 
 // Handle success case: update dashboard metadata (id, uid) and update the list
 // of plugins
-func (r *ReconcileGrafanaDashboard) manageSuccess(dashboard *grafanav1alpha1.GrafanaDashboard, status GrafanaResponse, hash string) error {
+func (r *ReconcileGrafanaDashboard) manageSuccess(dashboard *grafanav1alpha1.GrafanaDashboard) error {
 	msg := fmt.Sprintf("dashboard %v/%v successfully submitted",
 		dashboard.Namespace,
 		dashboard.Name)
-
 	r.recorder.Event(dashboard, "Normal", "Success", msg)
 	log.Info(msg)
-
-	dashboard.Status.UID = *status.UID
-	dashboard.Status.ID = *status.ID
-	dashboard.Status.Slug = *status.Slug
-	dashboard.Status.Phase = grafanav1alpha1.PhaseReconciling
-	dashboard.Status.Hash = hash
-	dashboard.Status.Message = "success"
-
 	r.config.AddDashboard(dashboard)
 	r.config.SetPluginsFor(dashboard)
 
-	return r.client.Status().Update(r.context, dashboard)
+	return r.client.Update(r.context, dashboard)
```

Comment:
I don't think you need to update the CR at all any longer, do you? I don't think you have any changes to the status or metadata sections (and you shouldn't be able to update the spec).